content/blog/prioritizing-the-grind/index.md

```diff
@@ -0,0 +1,111 @@
+
+In a teacher/student environment we have a win-win situation. Teaching someone is not only rewarding on its own, but the thing you're teaching is further solidified with a deeper understanding while you're teaching it. Someone learning something new and fulfilling is a powerful way to improve their morale while increasing the number of people that have that skill on your team.
+
+If before you had an individual quietly fixing something that constantly breaks, or needs manual intervention regularly, you can break this cycle by adding more people to the process. The person that held it all together gets help, sometimes this can prevent burnout or employee retention in a serious way. Bringing more people into a situation like this can also give you unexpected results: a process that wasn't automated gets automated away because fresh eyes see a similar problem from the past.
+
+Fostering this environment will empower your team to continue this collaboration without being told to do so. People will automatically jump in to help those that need it, or offer their support to teach those that want to learn. This starts to become a fun place to go to work every day. It creates a cycle of more capable and happy individuals which leads to the feeling of rampant creation anyone that's been on one of these teams will recognize immediately.
+
+There is an upper limit to how expensive collaboration is on a given team. Communication scales exponentially for every new person on a team which can cause everything to grind to a halt. There are also best judgement calls you will make regularly about critical fires that come up that take precedence, as long as this is not the norm this is totally normal. Efficiency is important, but not if it kneecaps your team every day away from what will help them grow and work on what they feel is important. 
+
+### Predictability before speed
+
+<!-- prettier-ignore-start -->
+<!-- CODE BLOCK - START -->
+```js
+do {
+  performWellDocumentedTaskList();
+} while (criticalInterruptTasks.length < 0)
+```
+<!-- CODE BLOCK - END -->
+<!-- prettier-ignore-end -->
+
+Prioritize consistency over moving as quickly as possible
+
+long term planning is impossible without predictability
+
+even in startup like situations, where speed is actually critical and not some arbitrary timeline, having predictability is a super power because you can maintain over the long term
+
+anti-crunch
+
+frequent checkpoints and demos
+
+### Outcome before activity
+
+<!-- prettier-ignore-start -->
+<!-- CODE BLOCK - START -->
+```js
+processes.filter(
+  (process) => !process.match(findBusywork)
+);
+```
+<!-- CODE BLOCK - END -->
+<!-- prettier-ignore-end -->
+
+Know why we're building the thing, share the why of how we're building the thing
+
+Flow vs red tape
+
+Counting points vs demoing features
```

Comment:
Are these points you'll elaborate on?

I think points vs demoing features could be a whole section

Reply:
Yeah the short-hand stuff is just for me to remember what I was thinking when I first thought of it. Most of the things I put up can be expanded, in fact most of my blog posts are just expansions of previous paragraphs or sentences from older ones...